core/src/main/java/smile/classification/LogisticRegression.java

```diff
@@ -75,7 +75,7 @@
  * 
  * @author Haifeng Li
  */
-public class LogisticRegression implements SoftClassifier<double[]>, Serializable {
+public class LogisticRegression implements SoftClassifier<double[]>, Serializable, OnlineClassifier<double[]> {
```

Comment:
Please put `OnlineClassifier` before `Serializable`.

Reply:
consider it done.